index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="topnav">
+        <a class="logo">My Logo</a>
+        <nav>
+            <div class="icon">
+                <div></div>
+                <div></div>
+                <div></div>
+            </div>
+        </nav>
+    </header>
+    <section class="headline">
+    <h1 class="name">
+        I'm Festy<br />
+        Glad to meet you!<br />
+    </h1>
+    <p class="words">
+        I'm a software developer! I can help you build a product,
+        feature or website. Look through some of my work and experience!
+        If you like what you see and have a project need coded, dont hesitate
+        to contact me.
+    </p>
+    <p class="connect">LET'S CONNECT!</p>
+    <ul class="social-media">
+        <li><a href="#"><img src="./Images/twitter.svg" alt="twitter icon"></a></li>
+        <li><a href="#"><img src="./Images/Linkedin.svg" alt="Linkedin icon"></a></li>
+        <li><a href="#"><img src="./Images/Mirror.svg" alt="Mirror icon"></a></li>
+        <li><a href="#"><img src="./Images/Github.svg" alt=""></a></li>
+        <li><a href="#"><img src="./Images/peace.svg" alt="peace icon"></a></li>
+    </ul>
+    </section>
```

Comment:
You did a great job applying all the changes requested by the previous reviewer! 💯 However, there are still minor problems:

- [ ] 🚧 Your background image is not showing due to a typo in the path, please look into it! 🔧 
- [ ] 🚧 The title's font size is still not correct, please update it to be `40px` according to the FIGMA design! 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/212172339-2eee3916-915e-426c-972f-9d8d88ccd1f3.png) | ![image](https://user-images.githubusercontent.com/77069011/212172225-3f32ba92-200e-40ea-b032-50d574c68352.png) |